bt2.txt

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#include <stdlib.h>
+
+void tong_hieu(int a, int b, float c, float d){
```

Comment:
Hi Sone,

Next time, please use English for all functions and variables.

Thanks.